src/components/ColorPicker.tsx

```diff
@@ -144,73 +147,67 @@ const ColorSwatch: FC<{
       onTouchStart={toggleTextColor}
       // only add mousedown to desktop, otherwise it will activate twice on mobile
       onMouseDown={!isTouch ? toggleTextColor : undefined}
-      style={{ cursor: 'pointer' }}
+      className={css({ cursor: 'pointer' })}
     >
       {shape === 'bullet' ? (
         <span
-          style={{
-            color,
+          className={css({
             display: 'inline-block',
-            fontSize: size,
             margin: '3px 5px 5px',
-            width: size - 1,
-            height: size - 1,
             textAlign: 'center',
-          }}
+          })}
+          style={{ color, fontSize: size, width: size - 1, height: size - 1 }}
         >
           •
         </span>
       ) : (
         <TextColorIcon
-          size={size}
-          style={{
-            backgroundColor,
-            border: `solid 1px ${selected ? colors.fg : 'transparent'}`,
+          cssRaw={css.raw({
+            border: selected ? `solid 1px {colors.fg}` : `solid 1px transparent`,
             fontWeight: selected ? 'bold' : 'normal',
-            color: backgroundColor ? 'black' : color,
             margin: '3px 5px 5px',
-          }}
+          })}
+          size={size}
+          style={{ color: backgroundColor ? 'black' : color, backgroundColor }}
         />
       )}
     </span>
   )
 }
 
 /** Text Color Picker component. */
-const ColorPicker: FC<{ fontSize: number; style?: React.CSSProperties }> = ({ fontSize, style }) => {
+const ColorPicker: FC<{ fontSize: number; style?: React.CSSProperties; cssRaw?: SystemStyleObject }> = ({
+  fontSize,
+  style,
```

Comment:
The `style` prop is no longer passed by the only component using `<ColorPicker>`. Can we remove it?

https://github.com/cybersemics/em/blob/3d25ccaa6270bf91b2f032fc3dc6d8079d97f4ae/src/components/icons/TextColorWithColorPicker.tsx#L43